src/bin/casr-san.rs

```diff
@@ -231,16 +231,23 @@ fn main() -> Result<()> {
     } else {
         // Get termination signal.
         if let Some(signal) = sanitizers_result.status.signal() {
+            // Get stack trace and mappings from gdb.
             match signal {
-                4 => {
+                4 | 31 => {
                     report.execution_class =
                         ExecutionClass::find("BadInstruction").unwrap().clone();
                 }
+                5 => {
+                    report.execution_class = ExecutionClass::find("TrapSignal").unwrap().clone();
+                }
                 6 => {
                     report.execution_class = ExecutionClass::find("AbortSignal").unwrap().clone();
                 }
-                11 => {
-                    report.execution_class = ExecutionClass::find("SEGV").unwrap().clone();
+                7 | 11 => {
+                    println!("Segmentation fault occured, but there is not enough information availibale to determine \
```

Comment:
Do we really need to print anything to console?

Reply:
--stdout option would fail, there will be additional lines to JSON

Reply:
println -> eprintln